test/compress_test.js

```diff
@@ -23,6 +23,8 @@ exports.compress = {
       actual.push(entry.path);
     });
     parse.on('close', function() {
+      actual.sort();
+      expected.sort();
```

Comment:
that said with the stat passing change, this may no longer be need but doesnt hurt anything imho.